contracts/strategies/dnsVectorStrategy/DnsVectorStrategyAumLib.sol

```diff
@@ -128,13 +126,9 @@ library DnsVectorStrategyAumLib {
             uint112 reserve0, // wAvax (generally it can be aaveSupplyToken or aaveBorrowToken) // usdc (generally it can be aaveSupplyToken or aaveBorrowToken)
             ,
 
-        ) = strategyStorage.traderJoePair.getReserves();
-        uint256 lpTokenTotalSupply = strategyStorage
-            .traderJoePair
-            .totalSupply();
-        uint256 lpTokenContractBalance = strategyStorage
-            .vectorPoolHelperJoe
-            .balanceOf(address(this));
+        ) = strategyStorage.pangolinPair.getReserves();
```

Comment:
why not call 'getPangolinLpReserve'? gas?

Reply:
Refactored LP related code.